### PR TITLE
Adding x-sap-odm-semantic-key

### PR DESCRIPTION
**Pull request by ArkadiyPonomarov**
A new SAP extension x-sap-odm-semantic-key needs to be introduced.

### CODE REVIEW

File: sap-schemas/v3.0/README.md [hunk 1]
```diff
@@ -335,6 +335,37 @@ Example:
 }
 ```
 
+### `x-sap-odm-semantic-key`
+
+- Type: `Array`
+- Used at: [Schema Object](https://spec.openapis.org/oas/v2.0#schema-object)
```
Comment (pavelkornev) [inline sap-schemas/v3.0/README.md:341]: Wrong spec URL. Should be:
https://spec.openapis.org/oas/v3.0.3#schema-object

File: sap-schemas/v2.0/README.md [hunk 2]
```diff
@@ -381,6 +381,37 @@ Example:
 }
 ```
 
+### `x-sap-odm-semantic-key`
+
+- Type: `Array`
+- Used at: [Schema Object](https://spec.openapis.org/oas/v2.0#schema-object)
+- Description: Semantic key(s).
```
Comment (pavelkornev) [inline sap-schemas/v2.0/README.md:388]: I think we need more detailed description here. 1-2 sentences.

**Review by pavelkornev — CHANGES_REQUESTED** [review]
There is no need to put the same field into 2 different schema files, you can use just `extensions.json`... previous ODM contributions were done slightly incorrectly.

File: sap-schemas/v3.0/README.md (hunk 1 above)
Comment (ArkadiyPonomarov) [reply sap-schemas/v3.0/README.md]: Fixed.

File: sap-schemas/v2.0/README.md (hunk 2 above)
Comment (ArkadiyPonomarov) [reply sap-schemas/v2.0/README.md]: Description has been changed.

**Review by pavelkornev — APPROVED** [review]

**Pull request merged**